jjb/include/pipelines/qpc-tests-pipeline.groovy

```diff
@@ -0,0 +1,529 @@
+    copyArtifacts filter: 'postgres.*.tar.gz', fingerprintArtifacts: true, projectName: 'qpc-testing-build', selector: lastCompleted()
+}}
+
+
+def getMasterQPC() {{
+    // Grabs the master qpc builds and sets it up
+    echo 'getMasterQPC'
+    def qpc_version = getQPCVersion()
+    getQuipucordsBuild()
+
+    sh 'ls -lah'
+    echo 'Extract the installer script'
+    sh """\
+    echo ${{qpc_version}}
+    tar -xvzf quipucords.${{qpc_version}}.install.tar.gz
+    """.stripIndent()
+
+    echo 'Copy container to installer packages directory'
+    sh '''\
+    mkdir -p install/packages
+    '''.stripIndent()
+
+}}
+
+
+def getReleasedQPC() {{
+    echo 'Get released QPC'
+    // Clean up QPC package if here...
+    sh '''\
+    ls -l
+    if [ -f quipucords.*tar.gz ]; then
+        rm quipucords.*.tar.gz
+    fi
+    '''.stripIndent()
+
+    // Pulls down Released Container
+    echo "load docker container from tarball"
+    sh "curl -k -O -sSL https://github.com/quipucords/quipucords/releases/download/{release}/quipucords.{release}.tar.gz"
+    // Pulls down Released Install
+    sh "curl -k -O -sSL https://github.com/quipucords/quipucords/releases/download/{release}/quipucords.{release}.install.tar.gz"
+
+    echo "extract the installer into ${{WORKSPACE}}/install"
+
+    sh "tar -xvzf ${{WORKSPACE}}/quipucords.{release}.install.tar.gz"
+}}
+
+
+
+///////////////////////////
+//// Install Functions ////
+///////////////////////////
+def installQPC(distro) {{
+    // Install QPC
+    if ('{install_type}' == 'nosupervisord') {{
+        installQPCNoSupervisorD()
+    }} else if ('{install_type}' == 'container') {{
+        containerInstall(distro)
+    }} else {{
+        defaultInstall()
+    }}
+    // Install the client
+}}
+
+
+def defaultInstall() {{
+    echo "Execute install.sh to install"
+    dir("${{WORKSPACE}}/install") {{
+        sh 'pwd'
+        sh 'ls -l'
+        sh 'sudo ./install.sh -e server_install_dir=${{WORKSPACE}}'
+    }}
+}}
+
+
+def installQPCNoSupervisorD() {{
+    echo "Execute install.sh to install without supervisord"
+    dir("${{WORKSPACE}}/install") {{
+        sh 'pwd'
+        sh 'ls -l'
+        sh 'sudo ./install.sh -e server_install_dir=${{WORKSPACE}} -e use_supervisord=false'
+
+        // Docker log to check for supervisord
+        sh 'sudo docker ps -a'
+        sh 'sudo docker logs quipucords | grep -i "Running without supervisord"'
+    }}
+}}
+
+
+def containerInstall(distro) {{
+    echo "Execute docker container install"
+    def qpc_version = getQPCVersion()
+    if (distro == 'rhel6') {{
+        echo 'No systemd, Docker should already be started.'
+    }} else {{
+        echo 'Starting Docker using Systemd'
+        sh 'sudo systemctl start docker'
+        sh 'sudo systemctl status docker'
+    }}
+
+    sh """\
+    sudo docker load -i quipucords.${{qpc_version}}.tar.gz
+    # make log dir to save server logs
+    mkdir -p log
+    """.stripIndent()
+}}
+
+
+def installQpcClient(distro) {{
```

Comment:
Other places you do camel case but keep `QPC` all caps. Better to be consistent for future refactor -- either works

Reply:
Good Call. Fixed it :)